recording-oracle/src/modules/liquidity-score/liquidity-score.controller.ts

```diff
@@ -43,4 +43,20 @@ export class LiquidityScoreController {
   ): Promise<UploadFile | null> {
     return await this.liquidityScoreService.calculateLiquidityScore(payload);
   }
+
+  @UseGuards(ApiKeyGuard)
```

Comment:
Using ApiKeyGuard here means that the frontend needs to have the api key, the same api key used to trigger calculateLiquidityScore and any user could obtain it.
This endpoint should be public.